src/main/kotlin/persistence/model/ILanguageEntity.kt

```diff
@@ -0,0 +1,24 @@
+package persistence.model
+
+import io.requery.*
+
+@Entity
+interface ILanguageEntity: Persistable {
+
+    @get:Key
+    @get:Generated
+    var id: Int
+
+    @get: Column(unique = true, nullable = false)
+    val slug: String
+    @get: Column(nullable = false)
+    val name: String
+    @get: Column(nullable = false)
+    val gateway: Boolean
```

Comment:
should we change this to isGateway for consistency with the language data class?

Reply:
I think I ran into requery trouble when we tried that, but I didn't look into much detail, so it'd be worth revisiting again.